arches/app/views/api.py

```diff
@@ -1457,6 +1457,31 @@ def post(self, request, tileid):
         return tileview.post(request)
 
 
+@method_decorator(csrf_exempt, name="dispatch")
+class NodeGroup(APIBase):
+    def get(self, request, nodegroupid=None):
+        params = request.GET.dict()
+        user = request.user
+        perms = "models." + params.pop("perms", "read_nodegroup")
+        params["nodegroupid"] = params.get("nodegroupid", nodegroupid)
+
+        try:
+            uuid.UUID(params["nodegroupid"])
+        except ValueError as e:
+            del params["nodegroupid"]
+
+        try:
+            nodegroup = models.NodeGroup.objects.get(pk=params["nodegroupid"])
+            permitted_nodegroups = [nodegroup.pk for nodegroup in get_nodegroups_by_perm(user, perms)]
+        except Exception as e:
+            return JSONResponse(str(e), status=404)
+
+        if not nodegroup or not nodegroup.pk in permitted_nodegroups:
```

Comment:
Isn't the result the same if it's written like this? I find the syntax clearer.
```not nodegroup or nodegroup.pk not in permitted_nodegroups```

Reply:
👍